types/react-select/src/utils.d.ts

```diff
@@ -68,10 +68,10 @@ export function getScrollParent(element: React.Ref<any>): Element;
 // ------------------------------
 
 export function animatedScrollTo(
-  element: Element,
+  element: HTMLElement | typeof window,
```

Comment:
IMO `Element` instead of `HTMLElement` was somewhat justified because while calling `scrollTo` on SVG nodes rarely makes sense, the function does still technically work with an `SVGElement` parameter.

Reply:
Fair.